<launch>
  <node respawn="true" pkg="joy" type="joy_node" name="turtle_joy">
    <param name="dev" type="string" value="/dev/input/js1" />
    <param name="deadzone" value="0.12" />
  </node>

  <param name="axis_linear" value="7" type="int"/>
  <param name="axis_angular" value="6" type="int"/>
  <param name="scale_linear" value="2" type="double"/>
  <param name="scale_angular" value="2" type="double"/>
  <node pkg="joystick_teleop" type="joystick_teleop" name="teleop"/>

	<node pkg="collision_avoidance" type="collision_avoidance" name="collision" output="screen">
    <param name="treshold_bottom" value="0.005" type="double"/>
    <param name="treshold_up" value="0.01" type="double"/>
    <param name="max_speed" value="0.5" type="double"/>

    <remap from="~/output_vel" to="/vrep/twistCommand" />
    <remap from="~/pointcloud" to="/vrep/hokuyoSensor" />
    <remap from="~/out_speed_from_joy" to="/out_speed_from_joy" />
    
    <!--<remap from="~/output_vel" to="/mobile_base/commands/velocity" />-->
    <!--<remap from="/mobile_base/sensors/bumper_pointcloud" to="~/pointcloud" />-->
  </node>
</launch>

